man/systemd-measure.xml

```diff
@@ -209,26 +216,31 @@
     <example>
       <title>Generate a unified kernel image, and calculate the expected TPM PCR 11 value</title>
 
-      <programlisting># ukify --output foo.efi \
-     --os-release @os-release.txt \
-     --cmdline @cmdline.txt \
-     --splash splash.bmp \
-     --devicetree devicetree.dtb \
+      <programlisting>$ ukify --output=vmlinux.efi \
+     --os-release=@os-release.txt \
+     --cmdline=@cmdline.txt \
+     --splash=splash.bmp \
+     --devicetree=devicetree.dtb \
      --measure \
      vmlinux initrd.cpio
 11:sha1=d775a7b4482450ac77e03ee19bda90bd792d6ec7
 11:sha256=bc6170f9ce28eb051ab465cd62be8cf63985276766cf9faf527ffefb66f45651
-11:sha384=1cf67dff4757e61e5a73d2a21a6694d668629bbc3761747d493f7f49ad720be02fd07263e1f93061243aec599d1ee4b4
-11:sha512=8e79acd3ddbbc8282e98091849c3530f996303c8ac8e87a3b2378b71c8b3a6e86d5c4f41ecea9e1517090c3e8ec0c714821032038f525f744960bcd082d937da
+11:sha384=1cf67dff4757e61e5...7f49ad720be02fd07263e1f93061243aec599d1ee4b4
+11:sha512=8e79acd3ddbbc8282...0c3e8ec0c714821032038f525f744960bcd082d937da
```

Comment:
We have unicode, let's use it.
```suggestion
11:sha384=1cf67dff4757e61e5…7f49ad720be02fd07263e1f93061243aec599d1ee4b4
11:sha512=8e79acd3ddbbc8282…0c3e8ec0c714821032038f525f744960bcd082d937da
```

Reply:
No, we don't use unicode ellipsis in man pages, because docbook (or something else in the pipeline) replaces the unicode ellipsis by three dots anyway. So we'd get the alignment off, and also it'd be confusing to have the ellipsis in the sources and end up with the manual thing in the output anyway.